base/setup/usetup/lang/it-IT.h

```diff
@@ -146,31 +146,31 @@ static MUI_ENTRY itITIntroPageEntries[] =
     {
         6,
         11,
-        "ReactOS � nello stadio di sviluppo Alpha, significa che non tutte le funzioni",
+        "ReactOS \x8A nello stadio di sviluppo Alpha, significa che non tutte le",
         TEXT_STYLE_NORMAL
     },
     {
         6,
         12,
-        "sono operative e cambia continuamente. � adatto solamente per",
+        "funzioni sono operative e cambia continuamente. \x8A adatto solamente per",
```

Comment:
`\x8A` would translate to `è` here and not `È`. You might be looking for `\xC3\x88`.

Reply:
Thanks. I would correct and notify.

Reply:
![immagine](https://user-images.githubusercontent.com/9625583/41543699-ab517f46-7317-11e8-8809-392d5a445f9b.png)
 
I've tried that but seems that somethins is wrong with capital letter encoding.

Reply:
There's some garbage after "della lista."

Reply:
Yes. Is me tring to implemente \xC3\x88 character. But it produce some garbage and not the right encoded char.

Reply:
Try using `\x98`.
(Note: é == `\x82`, è == `\x8A`, and É == `\x90`).

Reply:
`\x90` works fine but I'm not able to finde U with grave accent.

Reply:
I'm not sure why you're messing with all this \x stuff. All the other translations just put the actual character (in the respective encoding).

Reply:
I've tried but when I compile all special character where corrupted and also in the source code special char had wrong encoding.